src/core/css/SearchResult.scss

```diff
@@ -40,6 +49,44 @@
   }
 }
 
+.SearchResult-authors {
+  display: inline-block;
+  font-weight: normal;
+  margin: 0;
+  padding: 0;
+
+  &::after {
+    content: ' — ';
```

Comment:
Does this need to be localised? It looks like it's showing up in the default link blue/purple as well.